README.md

```diff
@@ -6,20 +6,23 @@
 
 # nrich
 
-Nrich is a Java library developed at CROZ whose purpose is to make development of applications on JVM a little easier. It contains modules that were found useful on projects and as such were extracted
-to a common library. It is based on Spring Framework it also provides Spring Boot starters for most of the modules to make configuration easier.
+Nrich is a Java library developed at CROZ whose purpose is to make development of applications on the JVM a little
+easier.
```

Comment:
Can you explain why this is in a new row (in raw format)?

Reply:
It turned it I had a weird markdown plugin for IDEA that recommended this line break.